API.md

```diff
@@ -23,10 +23,11 @@ import rers from "path/to/librers.so";
 
 ### Constructor `rers.init(INT cache_size = 1000)`
 
-Build a regex store, optionally specifying its size `n` (defaults to 100). The
+Build a regex store, optionally specifying its size `n` (defaults to 1000). The
```

Comment:
That's not a lint

Reply:
It was a minor mismatch of docs vs code, ie a lint 😀

Reply:
Ok, then that works
 I'd have opted for the docfix instead, but it probably doesn't matter much